app/src/main/java/org/greenstand/android/TreeTracker/messages/announcementmessage/AnnouncementScreen.kt

```diff
@@ -0,0 +1,249 @@
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.input.nestedscroll.nestedScroll
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.unit.dp
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.viewmodel.compose.viewModel
+import org.greenstand.android.TreeTracker.R
+import org.greenstand.android.TreeTracker.models.messages.AnnouncementMessage
+import org.greenstand.android.TreeTracker.root.LocalNavHostController
+import org.greenstand.android.TreeTracker.theme.CustomTheme
+import org.greenstand.android.TreeTracker.view.ActionBar
+import org.greenstand.android.TreeTracker.view.AppButtonColors
+import org.greenstand.android.TreeTracker.view.AppColors
+import org.greenstand.android.TreeTracker.view.ArrowButton
+import org.greenstand.android.TreeTracker.view.CustomDialog
+import org.greenstand.android.TreeTracker.view.LocalImage
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun AnnouncementScreen(
+    userId: Long,
+    otherChatIdentifier: String,
+    viewModel: AnnouncementViewModel = viewModel(
+        factory = AnnouncementViewModelFactory(
+            userId,
+            otherChatIdentifier
+        )
+    )
+) {
+    val scrollState = rememberLazyListState()
+    val scrollBehavior = remember { TopAppBarDefaults.pinnedScrollBehavior() }
+    val state by viewModel.state.observeAsState(AnnouncementState())
+    val navController = LocalNavHostController.current
+
+
+    Scaffold(
+        topBar = {
+            ActionBar(
+                leftAction = {
+                    Box(
+                        modifier = Modifier
+                            .align(Alignment.Center)
+                            .aspectRatio(1f)
+                            .fillMaxSize()
+                            .padding(
+                                top = 10.dp,
+                                bottom = 10.dp
+                            )
+                            .background(color = AppColors.MediumGray, shape = CircleShape),
+                        contentAlignment = Alignment.Center
+                    ) {
+                        Text(
+                            text = stringResource(id = R.string.admin_placeholder).uppercase(),
+                            color = CustomTheme.textColors.lightText,
+                            fontWeight = FontWeight.Bold,
+                            style = CustomTheme.typography.regular,
+                        )
+                    }
```

Comment:
I've created a composable for this. Please use either RoundedLocalImageContainer or RoundedImageContainer